<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadCurso</class>
 <widget class="QWidget" name="cadCurso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Cursos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/imagens/Imagens/logo.png</normaloff>:/imagens/Imagens/logo.png</iconset>
  </property>
  <widget class="QLabel" name="labelIcone">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/recursos/Imagens/Curso.png&quot; height=&quot;90&quot; width=&quot;90&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCadastrar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValid">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2" colspan="2">
     <widget class="QDoubleSpinBox" name="boxDuracao">
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="labelAnos">
      <property name="text">
       <string>anos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelNome">
      <property name="text">
       <string>Nome*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="campoIDCurso">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="labelDuracao">
      <property name="text">
       <string>Duração*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="campoNome">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="labelIdCurso">
      <property name="text">
       <string>ID do Curso:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>campoNome</tabstop>
  <tabstop>campoIDCurso</tabstop>
  <tabstop>btnCadastrar</tabstop>
  <tabstop>boxDuracao</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
